assets/section-main-product.css

```diff
@@ -653,7 +653,7 @@ a.product__text {
 
 .product__media-toggle:focus-visible:after {
   box-shadow: 0 0 0 0.3rem rgb(var(--color-background)),0 0 0rem 0.5rem rgba(var(--color-foreground), 0.5);
-  border-radius: var(--media-radius) - var(--media-border-width);
+  border-radius: calc(var(--media-radius) - var(--media-border-width));
```

Comment:
it's actually overwritten below on line 665 and from what I can tell it looks better with the border radius that matches the `media-radius` only 🤔 
[video](https://screenshot.click/17-21-ppgtj-8mcd5.mp4)

I'd suggest updating to:
```suggestion
  border-radius:var(--media-radius);
```

And then removing the declaration that's on line `664-666`

Reply:
Thanks for the feedback and the clear video!

I agree that having just the `--media-radius` as radius looks a lot better. After removing the duplicate `product__media-toggle:focus-visible:after` declaration we're left with two blocks with exactly the same properties:

![image](https://user-images.githubusercontent.com/5780600/168917610-512ba6eb-fdd4-4d3b-84e0-ea4509d602d8.png)

I've now pushed a commit implementing your suggestions and merging these blocks into one. For consistency maybe it's better to also merge the two identical blocks above it?

![image](https://user-images.githubusercontent.com/5780600/168918602-ddf42f07-03d0-4fa2-9a0f-66a9317ffcb1.png)

Reply:
Ah actually we can't have the two grouped together. It's not obvious at first but basically we're using the `.focused` class specifically for browsers that don't support `:focus-visible` yet. 
Safari only supports it in the latest `15.4` version and we're supporting the last 2 versions (`14` and `15`). 

When grouped what happens is, the browser sees the pseudo class and doesn't `understand` it so it skips the whole declaration event if the other part of it should be `understood`. 

All that to say that we didn't group them on purpose for browser support 🙂

Reply:
Aah that makes sense, reverted that change now